src/main/java/ru/javarush/quest/entity/Account.java

```diff
@@ -0,0 +1,64 @@
+package ru.javarush.quest.entity;
+
+public class Account {
+    private Long id;
+    private final String name;
+    private final String login;
+    private String password;
+
+    public Account(Long id, String name, String login, String password) {
```

Comment:
We have 4 fields. Let's make all fields final and use builder pattern

Reply:
What is the different will be if i add builder pattern here? and why i have to add just in Account class no in Answer or Question classes?